src/EditorFeatures/CSharpTest/Diagnostics/SpellCheck/SpellCheckTests.cs

```diff
@@ -461,5 +461,49 @@ void M()
 
             await TestAsync(text, expected, compareTokens: false);
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsSpellcheck)]
+        [WorkItem(13345, "https://github.com/dotnet/roslyn/issues/13345")]
+        public async Task TestMissingOnSnippet()
+        {
+            await TestAsync(
```

Comment:
The test name is that it's missing, but it's being replaced (presumably because the *keyword* version is still there).

Maybe use an example snippet that isn't also a keyword?